style.css

```
@@ -0,0 +1,79 @@
+* {
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+}
+
+body {
+	min-height: 100vh;
+
+	background-color: #000000;
+
+	display: flex;
+	justify-content: center;
+	align-items: center;
```

```suggestion
	display: grid;
	place-items: center;
```
you can do this task in 2 line